CustomApps/new-releases/index.js

```diff
@@ -40,10 +40,12 @@ const CONFIG = {
 	relative: getConfig("new-releases:relative", false)
 };
 
+let dismissed = [JSON.parse(Spicetify.LocalStorage.get("new-releases:dismissed")) || null].flat();
```

Comment:
Catch `JSON.parse` in case it throws an error, like in any other part of our codebase. Also since if it is already an array, you won't need to construct a new array out of it.
```suggestion
let dismissed;
try {
	dismissed = JSON.parse(Spicetify.LocalStorage.get("new-releases:dismissed");
	if (!Array.isArray(dismissed)) throw "";
} catch {
	dismissed = [];
}
```